Common/GTesting/itkImageMaskSpatialObject2GTest.cxx

```diff
@@ -16,6 +16,9 @@
  *
  *=========================================================================*/
 
+#include <itkConfigure.h> // For ITK_VERSION_MAJOR
+
+#if ITK_VERSION_MAJOR < 5
 
 // First include the header file to be tested:
 #include "itkImageMaskSpatialObject2.h"
```

Comment:
this does not work anymore if this file is deleted right?

Reply:
Good point @mstaring ! I think the entire ImageMaskSpatialObject2 test should be removed, unconditionally. It wasn't such an interesting test anyway. Just my first attempt to get GoogleTest working with elastix.